tests/rust/no_includes.rs

```diff
@@ -0,0 +1,4 @@
+#[no_mangle]
+pub extern "C" fn hello_world() {
```

Comment:
Could you rename this to `root` and drop the println!()? That would match our other tests.